test/e2e/mongodbtests/mongodbtests.go

```diff
@@ -200,7 +202,7 @@ func Connectivity(mdb *mdbv1.MongoDB, username, password string) func(t *testing
 			Username:      username,
 			Password:      password,
 		})); err != nil {
-			t.Fatal(fmt.Sprintf("Error connecting to MongoDB deployment: %+v", err))
+			t.Fatal(fmt.Sprintf("Error connecting to MongoDB deployment: %s", err))
```

Comment:
[q] is there a specific reason here we use `fmt.Sprintf`? A few lines below we have the same structure (`t.Fatal`) with inside a `errors.Errorf`